nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-restapi-processors/src/main/java/org/apache/nifi/processors/elasticsearch/AbstractJsonQueryElasticsearch.java

```diff
@@ -277,8 +281,8 @@ private void handleAggregations(final Map<String, Object> aggregations, final Pr
         }
     }
 
-    private FlowFile writeHitFlowFile(final int count, final String json, final ProcessSession session,
-                                      final FlowFile hitFlowFile, final Map<String, String> attributes) {
+    public FlowFile writeHitFlowFile(final int count, final String json, final ProcessSession session,
```

Comment:
Doesn't need to be `public`, package-private should be enough (i.e. remove the access modifier)

Reply:
👍🏻